analyzers/src/SonarAnalyzer.Common/AnalysisContext/SonarAnalysisContextBase.cs

```diff
@@ -125,4 +108,12 @@ public bool HasMatchingScope(DiagnosticDescriptor descriptor)
         bool ContainsTag(string tag) =>
             descriptor.CustomTags.Contains(tag);
     }
+
+    private ImmutableHashSet<string> CreateUnchangedFilesHashSet(AnalyzerOptions options) =>
```

Comment:
`options` parameter is not used.

```suggestion
    private ImmutableHashSet<string> CreateUnchangedFilesHashSet() =>
```